app/src/lib/editors/win32.ts

```diff
@@ -302,6 +302,17 @@ const editors: WindowsExternalEditor[] = [
     displayNamePrefix: 'PhpStorm',
     publisher: 'JetBrains s.r.o.',
   },
+  {
+    name: 'Android Studio',
+    // registryKeys: registryKeysForJetBrainsIDE('Android Studio')
+    // Android Studio is based on JetBrains IDEs but I am not sure what to put here?
+    registryKeys: [
+      LocalMachineUninstallKey('Android Studio'),
+    ],
+    executableShimPaths: executableShimPathsForJetBrainsIDE('studio'),
+    displayNamePrefix: 'Android Studio',
+    publisher: 'JetBrains s.r.o',
```

Comment:
Another small note the publisher should be `Google LLC`

Reply:
Yes, the publisher is Google LLC but it says JetBrains s.r.o in Task Manager when I open it

Reply:
Yes, it should match what you see in the registry keys from the screenshot you shared 😄